cloudbank-v3/spring-apps-spring3/README.md

```diff
@@ -0,0 +1,288 @@
+      ```
+
+1. Deploy transfer service
+
+   ```shell
+   oractl:>deploy --app-name cbv3 --service-name transfer --artifact-path transfer/target/transfer-0.0.1-SNAPSHOT.jar --image-version 0.0.1
+   uploading: transfer/target/transfer-0.0.1-SNAPSHOT.jarbuilding and pushing image...
+   creating deployment and service... successfully deployed
+   ```
+
+1. Deploy checks service
+
+   1. bind
+
+      ```shell
+      oractl:>bind --app-name cbv3 --service-name checks --username account
+      database password/servicePassword (defaults to Welcome12345): *************
+      Kubernetes secret for Datasource was created successfully.
+      ```
+
+   1. deploy
+
+      ```shell
+      oractl:>deploy --app-name cbv3 --service-name checks --artifact-path checks/target/checks-0.0.1-SNAPSHOT.jar --image-version 0.0.1
+      uploading: checks/target/checks-0.0.1-SNAPSHOT.jarbuilding and pushing image...
+      creating deployment and service... successfully deployed
+      ```
+
+1. Verify pods are running
+
+   `kgp -n cbv3`
+
+   ```text
+   NAME                           READY   STATUS    RESTARTS   AGE
+   account-5b68b6dbb6-l5sbd       1/1     Running   0          18m
+   checks-848c7c8898-cqpss        1/1     Running   0          36s
+   creditscore-5fd9f975c5-s5zlp   1/1     Running   0          11m
+   customer-665d64bbbd-mnm4h      1/1     Running   0          13m
+   testrunner-7cd7cb8f76-fg9ph    1/1     Running   0          4m10s
+   transfer-55c4664759-2tfvl      1/1     Running   0          7m33s
+   ```
+
+## Test Cloudbank
+
+1. Test account service
+
+   1. Port forward
+   `kpf -n cbv3 svc/account 8081:8080`
+   
+   1. Rest endpoint
+   `http --body :8081/api/v1/accounts`
+
+   ```json
+   [
+   {
+   "accountBalance": -20,
+   "accountCustomerId": "qwertysdwr",
+   "accountId": 149,
+   "accountName": "Andy's checking",
+   "accountOpenedDate": "2023-06-26T17:39:37.000+00:00",
+   "accountOtherDetails": "Account Info",
+   "accountType": "CH"
+   },
+   ...
+   ...
+   ```
+
+curl -i -X POST \
+-H 'Content-Type: application/json' \
+-d '{"journalType": "PENDING", "accountId": 149, "journalAmount": 100.00, "lraId": "0", "lraState": ""}' \
+http://localhost:8081/api/v1/account/journal
+
+```json
+{"journalId":21,"journalType":"PENDING","accountId":149,"lraId":"0","lraState":"","journalAmount":100}
+```
+
+`http --body :8081/api/v1/account/149/journal`
+
+```json
+[
+    {
+        "accountId": 149,
+        "journalAmount": 100,
+        "journalId": 21,
+        "journalType": "PENDING",
+        "lraId": "0",
+        "lraState": null
+    },
+    {
+        "accountId": 149,
+        "journalAmount": 100,
+        "journalId": 22,
+        "journalType": "PENDING",
+        "lraId": "0",
+        "lraState": null
+    },
+    {
+        "accountId": 149,
+        "journalAmount": 100,
+        "journalId": 23,
+        "journalType": "PENDING",
+        "lraId": "0",
+        "lraState": null
+    }
+]
+```
+
```

Comment:
throughout this file we need some text in between commands and output to explain what is going on